arrays-arnold-nate.js

```diff
@@ -0,0 +1,53 @@
+    // var groceryList = ["chips", "dip", "cookies"]
+// Write the code that will add "soda" to the end of the original array.
+    // groceryList.push("soda")
+    // console.log(groceryList)
+// Write the code that will add "granola" to the end of array without altering the original array.
+    // console.log(groceryList.concat("granola"))
+// Write the code that will return a subset of the array containing only "chips" and "dip".
+    // console.log(groceryList.slice(0, 2))
+// Write the code that will add "beans" to the "chips" and "dip" array.
+    // groceryList[2] = "beans"
+    // console.log(groceryList)
+// Consider the variable:
+
+ var numbers = [2, 4, 6, 8, 10]
+
+// Write the code that will add the number 0 to the beginning of the array.
+    // numbers.push("0")
+    // console.log(numbers.reverse())
+// Write the code that will add the number 12 to the end of the array.
+    numbers.push("12")
+    console.log(numbers)
+// Write the code that will remove the first number from the array.
+    console.log(numbers.shift())
+// Write the code that will add the number 0 to the beginning of the array without altering the original array. HINT: it's not .unshift You'll have to get creative! ;)
+    var number2 = [0]
+    console.log(number2.concat(numbers))
+    number2.push("0")
```

Comment:
Lines 25 and 26 are enough to answer this question. Line 27 places ) as a string at the end.